src/pages/index.tsx

```diff
@@ -1,75 +1,90 @@
-import type React from "react"
+import React from "react"
+import { data } from "@/constants"
+import { Link } from "@/components/shared/link"
+import { Image } from "@/components/shared/image"
 
 const Index: React.FC = () => {
   return (
-    <section className="span-max-w-4 md:span-max-w-8 xl:span-max-w-12 mx-auto pt-span-12">
-      <h1 className="text-4xl font-bold">Sizes</h1>
-      <Sizes />
-      <h1 className="text-4xl font-bold mt-span-12">Flex</h1>
-      <Flex />
-      <h1 className="text-4xl font-bold mt-span-12">Wrap</h1>
-      <Wrap />
-    </section>
-  )
-}
+    <section className="mx-[var(--grid-margins)] flex justify-center pt-span-12 uppercase">
```

Comment:
`mx-[var(--grid-margins)]` could also be added to tailwind config and used as `mx-grid-margin`, would look cleaner.